explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -505,21 +505,23 @@ private object SpectroscopyModesTable extends TableHooks:
       // atTop
       .useState(false)
       // Recalculate ITC values if the wv or sn change or if the rows get modified
-      .useStreamResourceBy((props, _, _, _, _, _, _, _, _, visibleRows, _) =>
```

Comment:
I think at one point the visible rows were sent to the itc first, but that is no longer used. I guess it was probably replaced by getting the rows via the table sort order.